yt/fields/astro_fields.py

```diff
@@ -36,11 +35,11 @@ def _dynamical_time(field, data):
                        units=unit_system["time"])
 
     def _jeans_mass(field, data):
-        MJ_constant = (((5.0 * pc.kboltz) / (pc.G * pc.mh)) ** (1.5)) * \
-          (3.0 / (4.0 * np.pi)) ** (0.5)
+        MJ_constant = (((5.0 * pc.kboltz) / (pc.G * pc.mh)) ** 1.5) * \
```

Comment:
Should we define this somewhere that it could be gotten at the top level?

Reply:
The usage is a bit obscure, not sure if it's worth it